.github/workflows/release.yml

```diff
@@ -176,46 +176,46 @@ jobs:
           --jf_cli_rt_name ${{ inputs.jf_cli_rt_name }} \
           --dry_run ${{ inputs.dry_run }} \
           --logging_level ${{inputs.logging_level }} \
-          --release_bundle_name ${{inputs.release_bundle_name }} \
-          --release_bundle_version ${{inputs.release_bundle_version }} \
-          --signing_key_name ${{ inputs.signing_key_name }} \
+          --release_bundle_name ${{ env.BUILD_NAME }} \
+          --release_bundle_version ${{ env.BUILD_NUMBER }} \
+          --signing_key_name "frog" \
           --arti_host ${{ env.ARTIFACTORY_HOST }} \
-          --release_method ${{ inputs.release_method }}
+          --release_method "artifacts"
       
-      - name: Push artifacts to Box
-        if: ${{ inputs.box_upload == true }}
-        shell: bash
-        run: |
-          python3 -m venv env
-          source env/bin/activate
-          pip3 install -q pyyaml dohq-artifactory requests boxsdk tqdm
+      # - name: Push artifacts to Box
```

Comment:
We are not to do Box upload anymore?

Reply:
the ticket said to remove the box upload parameters.

Reply:
Ok but this is all the code to do the upload itself!

Reply:
changes done!!